debezium-core/src/main/java/io/debezium/relational/RelationalChangeRecordEmitter.java

```diff
@@ -127,4 +144,8 @@ private void emitDeleteRecord(Receiver receiver, TableSchema tableSchema) throws
      * Returns the new row state in case of a CREATE or READ.
      */
     protected abstract Object[] getNewColumnValues();
+
+    protected boolean skipEmptyMessages() {
```

Comment:
Why is this needed?

Reply:
https://github.com/debezium/debezium/blob/master/debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/RecordsStreamProducer.java#L348

Reply:
Ok; a method comment would be good then (general advice for all API/SPI members).

Reply:
@gunnarmorling Added